gradle.properties

```diff
@@ -0,0 +1,7 @@
+kotlin.code.style=official
+org.gradle.parallel=true
+org.gradle.vfs.watch=true
+org.gradle.caching=true 
+org.gradle.configureondemand = true
+org.gradle.daemon=true 
+org.gradle.jvmargs=-Xmx4g 
```

Comment:
The default value of the heap size is 1GB, so can you tell me why you set it to 4GB?

Reply:
I wanted to give it enough size, and there's no particular reason. 